code/modules/mob/living/simple_animal/hostile/wumborian_fugu.dm

```diff
@@ -34,25 +42,21 @@
 	mob_size = MOB_SIZE_SMALL
 	environment_smash = ENVIRONMENT_SMASH_NONE
 	gold_core_spawnable = HOSTILE_SPAWN
-	var/wumbo = 0
-	var/inflate_cooldown = 0
-	var/datum/action/innate/fugu/expand/E
 	loot = list(/obj/item/fugu_gland{layer = ABOVE_MOB_LAYER})
-
-/mob/living/simple_animal/hostile/asteroid/fugu/Initialize()
-	. = ..()
-	E = new
-	E.Grant(src)
-
-/mob/living/simple_animal/hostile/asteroid/fugu/Destroy()
-	QDEL_NULL(E)
-	return ..()
+	///Whether or not the fugu is currently enlarged.
+	var/wumbo = FALSE
+	///The cooldown for the fugu's growth ability.
+	var/inflate_cooldown = 0
+	///How long the fugu's buffed form lasts.
+	var/inflate_time = 10 SECONDS
+	///How long it takes for the fugu to be able to grow again after shrinking.
+	var/reinflation_time = 8 SECONDS
 
 /mob/living/simple_animal/hostile/asteroid/fugu/Life(delta_time = SSMOBS_DT, times_fired)
-	if(!wumbo)
-		inflate_cooldown = max((inflate_cooldown - (0.5 * delta_time)), 0)
+	if(!wumbo && inflate_cooldown != 0)
+		inflate_cooldown = max(inflate_cooldown - (delta_time * 5), 0)
```

Comment:
What's with the `* 5` here? What does it represent?

Reply:
Inflate cooldown is now represented in seconds, and 1 delta_time = 1/2 a second, so in order to make the two work together, delta_time needs to be multiplied by 5 so we're working in the same units.